package.json

```diff
@@ -31,7 +31,7 @@
     "coverage": "ng test --code-coverage --single-run",
     "lint": "ng lint",
     "doc": "typedoc --out tsdocs --ignoreCompilerErrors src/",
-    "checkup": "concurrently -k \"npm run lint\"  \"npm run doc\" \"npm run build:prod\" \"npm run test\" \"npm run coverage\"",
+    "checkup": "npm run lint && npm run doc && npm run test && npm run coverage && npm run build:prod",
```

Comment:
Being a relative neophyte, does the `concurrently` not work as expected, or...?

Reply:
correct, concurrently was not working as expected.  Including test and coverage both trying to open the same port at the same time.  Removed for now, feel free to fix and add it back